vm/vmExpr.go

```diff
@@ -734,6 +734,27 @@ func invokeExpr(expr ast.Expr, env *Env) (reflect.Value, error) {
 	case *ast.CallExpr:
 		return callExpr(e, env)
 
+	case *ast.DeleteExpr:
+		mapExpr, err := invokeExpr(e.MapExpr, env)
+		if err != nil {
+			return nilValue, newStringError(e, err.Error())
+		}
+
+		keyExpr, err := invokeExpr(e.KeyExpr, env)
+		if err != nil {
+			return nilValue, newStringError(e, err.Error())
```

Comment:
This should be:
return nilValue, newStringError(e.KeyExpr, err.Error())

Reply:
done.